### PR TITLE
Setting up Prettier and ESLint with pre-commit hook

### LINKED ISSUE #5: install eslint, prettier, pre-commit hook

### PR DESCRIPTION
**Pull request by Rishav1707**
## This PR closes the following issue:
closes #5

### CODE REVIEW

**Review by huonggiangbui — CHANGES_REQUESTED** [review]
looks good! one thing -- can you remove typescript and its related packages, since we don't really use it? then i think it's good to go!

**Comment by Rishav1707** [comment]
> looks good! one thing -- can you remove typescript and its related packages, since we don't really use it? then i think it's good to go!

rm

File: frontend/.eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+module.exports = {
+  env: {
+    browser: true,
+    es2021: true
+  },
+  extends: [
+    'plugin:react/recommended',
+    'standard-with-typescript'
```
Comment (logonoff) [inline frontend/.eslintrc.js:8]: typescript isn't used in our project

**Review by logonoff — CHANGES_REQUESTED** [review]

**Comment by logonoff** [comment]
typescript is mentioned in package.json and other files when it is not used in the project

> > looks good! one thing -- can you remove typescript and its related packages, since we don't really use it? then i think it's good to go!
> 
> rm

**Pull request merged**